lambdas/question/src/main/java/uk.gov.di.ipv.cri.kbv.api.handler/QuestionHandler.java

```diff
@@ -43,83 +46,83 @@ public QuestionHandler() {
                                 KBVSessionItem.class,
                                 DataStore.getClient(
                                         ConfigurationService.getInstance().isRunningLocally())));
+        this.experianService = new ExperianService(this.objectMapper);
     }
 
-    public QuestionHandler(ObjectMapper objectMapper, StorageService storageService) {
+    public QuestionHandler(
+            ObjectMapper objectMapper,
+            StorageService storageService,
+            ExperianService experianService) {
         this.objectMapper = objectMapper;
         this.storageService = storageService;
+        this.experianService = experianService;
     }
 
     @Override
     @Tracing(captureMode = CaptureMode.DISABLED)
     public APIGatewayProxyResponseEvent handleRequest(
             APIGatewayProxyRequestEvent input, Context context) {
-
         String responseBody = "{}";
         Map<String, String> responseHeaders = Map.of("Content-Type", "application/json");
         String json;
         int statusCode;
 
         try {
             String sessionId = input.getHeaders().get(HEADER_SESSION_ID);
-            KBVSessionItem kbvSessionItem = storageService.getSessionId(sessionId);
+            KBVSessionItem kbvSessionItem =
+                    storageService.getSessionId(sessionId).orElseThrow(NullPointerException::new);
```

Comment:
this should probably result in a 400 error and not the server error we're giving it below